sui/docs/link-token.md

```diff
@@ -0,0 +1,533 @@
+4. User → ITS Source Chain: registerCustomCoin() → Deploys Token Manager A on Source Chain
+5. User → ITS Source Chain: linkCoin() → Deploys Token Manager B on Destination Chain
+6. User → Transfer TreasuryCap to the Token Manager (MINT_BURN type only)
+7. Token linking complete - InterchainTransfer enabled
+```
+
+## Prerequisites
+
+**Token Control**
+
+Before linking tokens, ensure you have Token permissions. Permission requirements depend on the manager type:
+
+- **LOCK_UNLOCK**: No token control or minter permissions required for that token
+    
+- **MINT_BURN**: You must have the [TreasuryCap](https://docs.sui.io/references/framework/sui_sui/coin#sui_coin_TreasuryCap) for the token on Sui to transfer it to the token manager
+
+## Token Manager Types
+
+For Sui, only the following token manager types are supported:
+
+- **LOCK_UNLOCK** (`2_u256`): For tokens that are locked/unlocked on the chain
+- **MINT_BURN** (`4_u256`): For tokens that are burned/minted on the chain
+
+**Important:** Linking two LOCK_UNLOCK tokens is not recommended. One token should be MINT_BURN (requiring `TreasuryCap` transfer if token deployed on Sui) and the other can be LOCK_UNLOCK (no permissions required). Using MINT_BURN on both sides is supported.
+
+## Parameters
+
+**Function Signature**
+
+```move
+public fun link_coin(
+    self: &InterchainTokenService,
+    deployer: &Channel,
+    salt: Bytes32,
+    destination_chain: String,
+    destination_token_address: vector<u8>,
+    token_manager_type: TokenManagerType,
+    link_params: vector<u8>,
+): MessageTicket { ... }
+```
+
+**Requirements**
+
+- `deployer`: An ITS `Channel` of the deployer of the coin as the sender's Sui address will not be tracked, only their `Channel` ID.
+- `salt`: Unique identifier for the token linking operation that will be used to generate a unique `tokenId`. On Sui, the `salt` must be 64 characters (e.g. 32 bytes), matching the Sui address format.
+- `destination_chain`: Name of the destination chain.
+- `destination_token_address`: Address of the token on the destination chain.
+- `token_manager_type`: Token manager type on Sui (e.g., `2_u256` for LOCK_UNLOCK or, `4_u256` for MINT_BURN)
+
+**Optional**
+- `link_params`: Bytes representation of an address on the destination chain that will be `Operator` of the destination token. Or, use an empty `vector` for no `Operator`.
+
+## Channel Role & Security
+
+#### `Channel` & Sender Identity
+
+Sui uses an object ownership model for security. Contracts cannot access sender parameters (such as the sender's Sui address), but capabilities can be granted by owning or transferring an object that posses the permissions required.
+
+In the context of ITS, the `Channel` object is used in place of the sender's Sui address managing, deploying and transferring tokens.
+
+#### Notable Entities Using `Channel`
+
+1. **Operator:** controls operational settings for a token manager, including flow limits and pausing.
```

Comment:
By pausing did you mean setting the flow limit to near 0? Or is this a different feature?